internal/platform/cli/cli.go

```diff
@@ -127,6 +127,8 @@ func Parse(args []string) (Config, error) {
 	fs.Alias("ste", "stream-errors")
 	fs.BoolVar(output, &config.StreamMatches, "stream-matches", true, "If specified, those requests that caused a match are printed to stdout during the scan (live)\n\tEnabled by default, can be disabled with --stream-matches=false or -stm=false")
 	fs.Alias("stm", "stream-matches")
+	fs.BoolVar(profile, &config.Poc, "poc", false, "Print only URLs")
+	fs.Alias("poc", "only-poc")
```

Comment:
If you look at the examples above, the "alias" is a shorter version of the original one. So, if "poc" is equivalent to "only-poc", the main flag must be defined as "only-poc".

```suggestion
	fs.BoolVar(profile, &config.Poc, "only-poc", false, "Print only URLs")
	fs.Alias("poc", "only-poc")
```

Reply:
done